.spi.yml

```diff
@@ -3,15 +3,10 @@ builder:
   configs:
   - platform: ios
     scheme: ComposableArchitecture
-    swift_version: 5.9
   - platform: macos-xcodebuild
     scheme: ComposableArchitecture
-    swift_version: 5.9
   - platform: tvos
     scheme: ComposableArchitecture
-    swift_version: 5.9
   - platform: watchos
     scheme: ComposableArchitecture
-    swift_version: 5.9
   - documentation_targets: [ComposableArchitecture]
-    swift_version: 5.9
```

Comment:
This instance of `swift_version` was only necessary to make docs being built with 5.9 when 5.8 was the default. By removing it, docs will still be built with 5.9 and will automatically move to use 5.10 etc once those become available.